wcfsetup/install/files/lib/system/form/builder/field/IconFormField.class.php

```diff
@@ -18,7 +18,7 @@ class IconFormField extends AbstractFormField implements IImmutableFormField
 {
     use TImmutableFormField;
 
-    private FontAwesomeIcon $icon;
```

Comment:
This one appears to be correct, but does not belong in this PR, because this is not PHPDoc.

There might also be some bug lurking here, because the `$icon` is not invalidated when the value changes. Perhaps the property should be removed entirely.